vyper/compiler/output_bundle.py

```diff
@@ -12,6 +12,7 @@
 from vyper.compiler.settings import Settings
```

Comment:
for verification, storage layout override can (and should) affect integrity hash.